frontend_tests/casper_tests/00-realm-creation.js

```diff
@@ -0,0 +1,91 @@
+var common = require('../casper_lib/common.js').common;
+
+var email = 'alice@example.com';
+var domain = 'example.com';
+var organization_name = 'Awesome Organization';
+
+casper.start('http://localhost:9981/create_realm/');
```

Comment:
@timabbott I have skipped `common.initialize_casper`. Including that creates $ not found while retrieving the JSON file. The login.js initliazes casper again with `common.initialize_casper`. So I hope that's okay. I have also made the test for realm creation as first one.